src/EditorFeatures/TestUtilities/Workspaces/TestHostProject.cs

```diff
@@ -362,7 +362,7 @@ private string GetTestOutputFilePath(string filepath)
 
             try
             {
-                outputFilePath = Path.GetDirectoryName(_filePath);
+                outputFilePath = Path.GetDirectoryName(filepath);
```

Comment:
Can we make this either a static method?  Why can't it use the field instead?

Reply:
it uses field such as Compilation, so can't be static. I don't know why it accepts filepath rather than not accepting any parameter and use all fields inside. but regardless, getting filepath is not a bug and more flexible if one want to reuse this method for different file path. so I am leaving that as it is.